cips/cip-noindex.md

```diff
@@ -0,0 +1,44 @@
+
+## Specification
+
+Remove the blobStartIndex from the ['WrappedTransaction'](https://celestiaorg.github.io/celestia-app/specs/data_structures.html#wrappedtransaction).
+
+## Rationale
+
+The blobStartIndex had two initial purposes:
+
+1. It was needed to create compact fraud proofs for blob inclusion proofs.
+2. It was needed to create the square out of the transactions alone in the absence of any constraints on blob placement.
+
+We are solving the first by committing to proving the correctness of the square layout in the future using a ZK proof, thereby removing the need for the blobStartIndex for fraud proofs.
+
+The second initial purpose was removed when we moved to a [deterministic square construction](https://github.com/celestiaorg/celestia-app/blob/main/docs/architecture/adr-020-deterministic-square-construction.md). With that the blobStartIndex was no longer needed for blob placements as it can be deterministically derived from the list of transactions. Each blob has to follow the [blob share commitment rules](https://celestiaorg.github.io/celestia-app/specs/data_square_layout.html#blob-share-commitment-rules) and cannot be placed at an index that does not respect the rules.
+
+## Backwards Compatibility
+
+Removing the index is a breaking consensus change but should not affect anybody upstream of this change.
+
+celestia-node does not consume this index. Although they have their own way of creating the same information, this proposal is not applicable to them.
+
+None of the rollup teams are influenced by this change except Sovereign SDK. The circuit parsing the PFB reserved namespace would break and must be adapted. The circuit does not use the information from the blobStartIndex. If this change is accepted, the live rollup teams will have to upgrade their circuits when Celestia upgrades to the new version. Currently, no rollups are live on mainnet using Celestia, so a breaking change would not affect anyone directly.
+
+## Security Considerations
+
+No Celestia light nodes rely on the blobStartIndex to verify the square's correctness. No fraud proofs rely on the blobStartIndex, so removing it does not affect the network's security. Without the blobStartIndex, we won't be able to create compact fraud proofs anymore. This means that accepting this proposal is also a commitment to ZK prove the validity of the square layout.
```

Comment:
Perhaps this can be moved to the forum discussion but has there been any analysis done on the feasibility of proving that a blob has been paid for

Reply:
Could you explain what you mean